### DISCUSSION TITLE
Spam shouldn't be required to click every time

### DISCUSSION THREAD

**Discussion by nguyenhoangan-monkey** [LAION-AI/Open-Assistant#3472] [Ideas]
The button spam shouldn't be clicked every time you want to say No.

**Comment by blankster** [comment]
I second this. It's the rare exception that something needs to be flagged as spam, while it's quite annoying to be forced to click every time on no, the majority of all times.

**Comment by mikayahlevi** [comment]
It should definitely be clear that it defaults to no, if this is implemented. No could just be highlighted before choosing an option.

**Comment by Dryhb** [comment]
Maybe a default selction of the rating, kind of similar thing as (almost) always clicking Polite and Harmless which are quite rare to find I experienced (any that would be rude or violent)
